### PR TITLE
Update tag for AVS SDK v1.20.1

### PR DESCRIPTION
**Pull request by lucianomartin**

### CODE REVIEW

**Comment by oscarbailey-xmos** [comment]
Just testing this now:
- Step 5 on the README of this page points to an old link, could be good to update it alongside these changes too

**Comment by oscarbailey-xmos** [comment]
Tested on my setup with a fresh Raspbian 2018/10, all working

**Review by oscarbailey-xmos — APPROVED** [review]

**Review by DaveAtkinsonXMOS — APPROVED** [review]
Tested, works for me.

File: auto_install.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ SETUP_DIR="$( pwd )"
 RPI_SETUP_DIR=$SETUP_DIR/vocalfusion-rpi-setup
 
 RPI_SETUP_TAG="v2.1.0"
-AVS_DEVICE_SDK_TAG="v1.13"
+AVS_DEVICE_SDK_TAG="xmos_v1.20.1"
```
Comment (mbanth) [inline auto_install.sh:7]: Change to the final tag once it has been created.

**Review by mbanth — APPROVED** [review]

**Comment by mbanth** [comment]
Looks like there's a conflict with auto_install.sh

**Comment by DaveAtkinsonXMOS** [comment]
This repo is to be discontinued so I am closing this pull request. Please refer to the new unified xmos/vocalfusion-avs-setup repo if you have any futher issues. Thanks.

**Pull request closed without merging**